pkg/kyverno/apply/command_test.go

```diff
@@ -0,0 +1,106 @@
+package apply
+
+import (
+	"bytes"
+	"io/ioutil"
+	"strings"
+	"testing"
+)
+
+func Test_ApplyCommandPass(t *testing.T) {
+	cmd := Command()
+	b := bytes.NewBufferString("")
+	cmd.SetOut(b)
+	cmd.SetArgs([]string{"../../../samples/best_practices/disallow_latest_tag.yaml", "-r", "../../../test/resources/pod_with_version_tag.yaml", "--policy-report"})
+	cmd.Execute()
+	actual, err := ioutil.ReadAll(b)
+	if err != nil {
+		t.Fatal(err)
+	}
+	expected := `
```

Comment:
Can we avoid comparing the output with the static expected result? It makes unit tests hard to maintain if there are any changes to the sample policy. 

For example, this message `- message: 'Validation error: CPU and memory resource requests and limits are required;` can be changed if we update sample policy.

We can deserialize `actual` to policyReport object, and check failed / success rules in `result`, summary count, etc.

Reply:
got it. I will make the changes.